src/NSwagStudio/NSwagStudio.csproj

```diff
@@ -15,6 +15,21 @@
     <WarningLevel>4</WarningLevel>

```

Comment:
Sorry. It wasn't supposed to have changed.

Dropped the change and rebased from current master.